app/templates/main/error.hbs

```diff
@@ -1,7 +0,0 @@
-{{#if slug}}
```

Comment:
I like the new look, but the text looks hard to notice below the image, could you try making it bigger and adding a more space between them?